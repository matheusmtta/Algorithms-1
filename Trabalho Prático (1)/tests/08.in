8 i
11 i
25 p
17 i
24 i
17 r
31 i
29 i
1 i
20 i
29 r
4 i
1 r
31 r
11 r
80 p
100 p
36 p
112 p
48 p
5 i
65 p
18 i
44 p
19 i
4 r
20 r
25 i
23 i
67 p
33 i
26 i
32 i
59 p
16 p
26 r
45 p
3 i
14 p
19 r
51 p
8 r
57 p
16 i
16 r
23 r
27 i
10 i
2 i
42 p
21 i
23 p
49 p
21 r
30 i
49 p
28 p
28 i
30 r
12 i
28 r
45 p
12 r
2 r
23 p
9 i
13 i
7 i
22 i
29 p
57 p
