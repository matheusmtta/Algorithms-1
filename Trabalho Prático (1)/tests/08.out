4
4
5
3
5
2
4
2
3
2
2
3
2
2
2
2
2
2
2
2
2
2
2
2
